webrtc.html

```diff
@@ -509,33 +509,42 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
           </li>
 
           <li>
-            <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            <p>Append an <code>operation</code> to
+            the <var>operations</var> queue, where
+            an <code>operation</code> represents the asynchronous
+            execution of the called function that will fulfill the
+            promise returned by the function.</p>.
```

Comment:
I reworded it to be modeled more like it was with ("stuff on top of the function").  I think that will make it more clear.  Please take another look.